assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -89,12 +89,24 @@ function useUploadMedia({ media, setMedia }) {
 
       try {
         const uploadedFiles = await Promise.all(
-          localFiles.map(async (localFile) => ({
-            ...localFile,
-            fileUploaded: getResourceFromAttachment(
+          localFiles.map(async (localFile) => {
+            trackEvent('upload_media', 'editor', '', '', {
+              type: localFile.file.type,
```

Comment:
I think we should add file size as well here.

Reply:
SGTM

Let's use `file_type` and `file_size` as keys then.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/6188/commits/f7d8a62cbb850d53ff2738c5aad84652eb34f302